### DISCUSSION TITLE
Getting more info from CoinGecko

### DISCUSSION THREAD

**Discussion by dongodao** [taoteh1221/Slideshow_Crypto_Ticker#41] [General]
Thanks for adding CoinGecko into your awesome project. I'm trying to add more info using the api:

https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=bitcoin

I've changed in functions.js line 1180: 
api[exchange] = 'https://api.coingecko.com/api/v3/simple/price?ids=' + ids + '&vs_currencies=' + currencies + '&include_24hr_vol=true';
to
api[exchange] = 'https://api.coingecko.com/api/v3/coins/markets?vs_currency=' +currencies+ '&ids='+ids;

And I have tried changing lines 874 and 875 about price_raw and volume_raw to all sorts of combination but it won't work. The lcd only shows:

BTC (Loading)

Your help would be appreciated. Thanks

**Comment by taoteh1221** [comment]
Try looking at the coingecko API documentation.
